### PR TITLE
[Access Monitoring] Add nav title & packages

### PR DESCRIPTION
**Pull request by ryanclark**
Adds the packages needed for the bar charts and query editor in Access Monitoring.

Also adds the nav title.

### CODE REVIEW

**Review by gzdunek — APPROVED** [review]

File: web/packages/shared/package.json [hunk 1]
```diff
@@ -10,11 +10,17 @@
...
+    "@nivo/bar": "^0.83.0",
     "@react-spring/web": "^9.7.2",
+    "@uiw/codemirror-theme-dracula": "^4.21.9",
+    "@uiw/react-codemirror": "^4.21.9",
     "@use-gesture/react": "^10.2.27",
     "ace-builds": "1.4.6",
     "create-react-class": "^15.6.3",
     "cross-env": "5.0.5",
+    "d3-time-format": "^4.1.0",
+    "d3-scale": "^4.0.2",
```
Comment (ryanclark) [inline web/packages/shared/package.json:23]: Should these go here or in `e-imports`? I imagine we'll probably end up using CodeMirror and graphs in OSS _at some point_, but I'm fine to move it

Comment (gzdunek) [reply web/packages/shared/package.json:23]: Yeah, if these are used only in the enterprise version, I would move them to `e-imports`.

**Review by kimlisa — APPROVED** [review]

File: web/packages/teleport/src/types.ts [hunk 2]
```diff
@@ -50,6 +50,7 @@ export enum NavTitle {
   // Access Management
   Users = 'Users',
   Roles = 'User Roles',
+  AccessMonitoring = 'Access Monitoring',
```
Comment (kimlisa) [inline web/packages/teleport/src/types.ts:53]: @ryanclark btw, access monitoring needs to go under the new section i just merged `Identity Governance & Security` line 60

File: web/packages/e-imports/package.json [hunk 3]
```diff
@@ -4,7 +4,14 @@
...
   "dependencies": {
+    "@codemirror/lang-sql": "^6.5.2",
+    "@nivo/bar": "^0.83.0",
     "@stripe/react-stripe-js": "^1.16.5",
-    "@stripe/stripe-js": "^1.48.0"
+    "@stripe/stripe-js": "^1.48.0",
+    "@uiw/codemirror-themes": "^4.21.20",
+    "@uiw/react-codemirror": "^4.21.20",
+    "d3-time-format": "^4.1.0",
+    "d3-scale": "^4.0.2",
```
Comment (ravicious) [inline web/packages/e-imports/package.json:14]: Is there any way we can update @nivo/bar and d3-scale to get rid of the vuln alert for d3-color?

https://github.com/gravitational/teleport/security/dependabot/211

**Pull request merged**